express-delete/index.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const app = express();
+
+let grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.use('/api/grades/:id', (req, res) => {
+  const { id } = req.params;
+  const result = Object.values(grades).filter(el => el.id !== Number(id));
+  grades = result;
+  res.send('success');
+});
+
+app.use('/api/grades', (req, res) => {
+  res.json(grades);
```

Comment:
This should respond with an array of `grade` objects. You need to use a `for in` loop to build the array and then send the array back to the client.

Reply:
![Screen Shot 2022-08-31 at 11 20 41 PM](https://user-images.githubusercontent.com/44100814/187845613-3434591c-fdc7-4612-88cb-e3a83c565ae0.png)